R/get_age_population_function.R

```diff
@@ -1,42 +1,45 @@
 #' return an interpolating function for populations in 1y age increments
 #'
 #' Return an interpolating function to get populations in 1y age increments
-#' from chunkier distributions produced by socialmixr::wpp_age() (must contain
-#' `lower.age.limit` and `population`)
+#' from chunkier distributions produced by socialmixr::wpp_age()
 #'
-#' @param population PARAM_DESCRIPTION
+#' @param population population data
+#' @param age_col age variable 
 #' @return OUTPUT_DESCRIPTION
 #' @details DETAILS
 #' @examples
 #' \dontrun{
-#' if (interactive()) {
-#'   # EXAMPLE1
-#' }
+#' get_age_population_function(population, age_col = lower.age.limit)
 #' }
 #' @export
-get_age_population_function <- function(population) {
+get_age_population_function <- function(population, 
+                                        age_col=lower.age.limit) {
   
```

Comment:
I remember you had made some changes to this, using `.data`, but I can't seem to find where those changes are in the other PRs